content/tokio/topics/tracing-next-steps.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Next steps with Tracing"
+---
+
+# Tokio-console
+
+Tokio-console is an htop-like utility that enables you to see a real-time view
+of an application’s spans and events. It can also represent "resources" that the
+Tokio runtime has created, such as Tasks. It's essential for understanding
+performance issues during the development process.
+
+To enable the console in mini-redis, you need to enable the `tracing` feature
+for the Tokio package:
+
+```toml
+# Update the tokio import in your Cargo.toml
+tokio = { version = "1", features = ["full", "tracing"] }
```

Comment:
This requires enabling `tokio_unstable` as well, right? We probably should explain *why* it is unstable as well.

Reply:
Gotcha, will update

Reply:
I added some explanation further below now!